### PR TITLE
Update docs for Solr 9.0

### PR DESCRIPTION
**Pull request by HoustonPutman**
@janhoy

Note: Now that all "supported" images aren't affected by the CVEs, we might be able to take that down...

### CODE REVIEW

**Comment by gerlowskija** [comment]
LGTM, fwiw.

File: solr/content.md [hunk 1]
```diff
@@ -26,12 +18,22 @@ $ docker run -p 8983:8983 -t %%IMAGE%%
...
 
-This repository is available on [github.com/docker-solr/docker-solr](https://github.com/docker-solr/docker-solr), and the official build is on the [Docker Hub](https://hub.docker.com/_/solr/).
+This repository is available on [github.com/apache/solr-docker](https://github.com/apache/solr-docker), but the image is built and maintained in the official Solr repo [github.com/apache/solr](https://github.com/apache/solr).
+
+Please direct any usage questions to the [Solr users mailing list](https://solr.apache.org/community.html#mailing-lists-chat).
 
 # History
 
-This project was started in 2015 by [Martijn Koster](https://github.com/makuk66). In 2019 maintainership and copyright was transferred to the Apache Lucene/Solr project. Many thanks to Martijn for all your contributions over the years!
+This project was started in 2015 by [Martijn Koster](https://github.com/makuk66) in the [github.com/docker-solr/docker-solr](https://github.com/docker-solr/docker-solr) repository. In 2019 maintainership and copyright was transferred to the Apache Solr project. Many thanks to Martijn for all your contributions over the years!
```
Comment (anshumg) [inline solr/content.md:29]: In 2019 maintainership -> In 2019, the maintainership ?

P.S: Not sure if 'maintainership' is a word but well, same is the case with 'committership'. I'm fine with the use of the word here. :)

File: solr/content.md [hunk 2]
```diff
@@ -26,12 +18,22 @@ $ docker run -p 8983:8983 -t %%IMAGE%%
...
+
+References:
+
+-	[CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228): Solr *was* vulnerable to this.
+-	[CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046): Solr *never was* vulnerable to this.
+-	[Solr's security bulletin](https://solr.apache.org/security.html#apache-solr-affected-by-apache-log4j-cve-2021-44228)
```
Comment (anshumg) [inline solr/content.md:39]: `Solr security bulletin` (remove the 's )

**Review by anshumg — COMMENTED** [review]
Don't have access to the repo so can't 'approve' the request but looks good to me.

**Pull request merged**